app/src/main/java/com/geeksville/mesh/MainActivity.kt

```diff
@@ -78,7 +80,15 @@ class MainActivity :
     private var showAppIntro by mutableStateOf(false)
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        enableEdgeToEdge()
+        enableEdgeToEdge(
+            // Disable three-button navbar scrim on pre-Q devices
+            navigationBarStyle = SystemBarStyle.auto(Color.TRANSPARENT, Color.TRANSPARENT),
+        )
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+            // Disable three-button navbar scrim
+            window.setNavigationBarContrastEnforced(false)
+        }
```

Comment:
The nav bar scrim generally looks bad and is only really suggested when above scrolling content. Disabling the scrim is [recommended ](https://developer.android.com/design/ui/mobile/guides/foundations/system-bars#button_modes)when the app has persistent bottom/nav bars.